pkg/chain/ethlike/generator/template/resubscribe.go.tmpl

```diff
@@ -0,0 +1,59 @@
+// Code generated - DO NOT EDIT.
```

Comment:
We were placing all of the templates in `tools/generators` so far, now we are breaking this layout. It is unclear to me if this is something we have reasons for? (cc @Shadowfiend)

Reply:
The main motivation for placing those files in `pkg/chain/ethlike/generator` was the fact they are very specialized ETH-like tools that are used only by ETH-like chain implementations. They are not as generic and ubiquitous as tools placed in `tools/generators`. This is why I felt right to keep them in the `ethlike` package close to other tools.

Reply:
Hello, I'm super-late here! `tools/` is meant for executable tools. Something in `tools/` can use library code in `pkg/` if needed, but if we're talking about a code generation tool, it belongs in `tools/`. We should never `go run` something in `pkg/`.